src/rshapes.c

```diff
@@ -197,19 +198,37 @@ void DrawLineBezier(Vector2 startPos, Vector2 endPos, float thick, Color color)
     Vector2 previous = startPos;
     Vector2 current = { 0 };
 
+    Vector2* pts = RL_MALLOC((2*BEZIER_LINE_DIVISIONS+2)*sizeof(Vector2));
```

Comment:
I prefer to avoid memory allocations. We can just use a stack allocated array:
```c
Vector2 points[2*BEZIER_LINE_DIVISIONS + 2] = { 0 };
```

Reply:
Fixed.